src/models/verification.ts

```diff
@@ -1,4 +1,14 @@
-export type VerificationSubject = Record<string, unknown>;
+export type VerificationSubjectData = Record<string, unknown>;
+
+export class VerificationSubject {
+  public constructor(data: Record<string, unknown>) {
+    this.data = data;
+    this.identifier = Symbol("verification-subject");
+  }
```

Comment:
When we download results, we read the input file (sequentially), then we can check if there is a decision associated to the row by using the symbol.

Because we read the file sequentially, we can ensure order, even when decisions are made out of order

This might become a problem with random ordering once we implement https://github.com/ecoacoustics/web-components/issues/75, but I'd say that the scope of order tracking is limited to that issue

_Side note_: I found the bug "Fixes a bug where the first page of a verification grid would really be the second" because the order was maintained in the results (I saw that the first 8 rows did not have a decision applied)

Reply:
OK, we probably need an index tracking variable for order - that will be more reliable. But you're right, out of scope for now